docs/guide/snaps-exports.md

```diff
@@ -0,0 +1,136 @@
+# Exports
+
+::: tip Developer Preview Software
+Snaps is pre-release software. To try Snaps, install [MetaMask Flask](https://metamask.io/flask).
+:::
+
+::: tip Feature Requests
+Do you have feature requests? Other ideas? We'd love to hear about them! [Click here](https://github.com/MetaMask/snaps-skunkworks/discussions) to join the discussion.
+:::
+
+## Table of Contents
+
+[[toc]]
+
+## `onRpcRequest`
+
+In order to communicate with dApps and other snaps, the snap must implement its own JSON-RPC API by exposing an exported function called `onRpcRequest`. Whenever the snap receives a JSON-RPC request, the `onRpcRequest` handler function will be called with the below parameters.
```

Comment:
In line 20 we use 'dapp' (uncapitalised) -- suggest you decide on one for consistency